jwst/mrs_imatch/mrs_imatch_step.py

```diff
@@ -337,17 +347,17 @@ def _match_models(models, channel, degree, center=None, center_cs='image'):
     # Therefore use sigma-clipping to detect and remove sources
     # (and unmasked outliers) prior to doing the background matching.
     # Loop over input exposures
-    for image, mask in zip(image_data, mask_data):
+    for image, mask in zip(image_data, mask_data, strict=False):
```

Comment:
here and below, do we expect these to have different length? if not, should we make it True?

Reply:
The ruff thingy must have added this because I have no recollection of putting `strict=False` here. I also don't know the answer. Given the step is deprecated, maybe we want to keep the backward compatible option, which I guess is `strict=False`?